Update Karmada CLI installation guide with detailed steps and dividing the setup into small interactive steps (#21)
karmada-io/playground: karmada-CLI-installation-example/step1/text.md

### Environment overview

The environment consists of two hosts:

> 1. `controlplane`: A host Kubernetes cluster is running, on which Karmada will be deployed. The kubeconfig files for the host cluster are located in the `$HOME/.kube` directory.

Reviewer: ```suggestion
1. `controlplane`: The host Kubernetes cluster where Karmada runs. The kubeconfig files for the host cluster are located in the `$HOME/.kube` directory.
```
